_includes/shared/markup/ccloud/get-ccloud.adoc

```diff
@@ -1,2 +1,2 @@
 This tutorial has some steps for Kafka topic management and/or reading from or writing to Kafka topics, for which you can use the Confluent Cloud Console or install the Confluent Cloud CLI.
-Instructions for installing Confluent Cloud CLI and configuring it to your Confluent Cloud environment is available from within the Confluent Cloud Console: navigate to your Kafka cluster, click on the `CLI and tools` link, and run through the steps in the `CCloud CLI` tab.
+Instructions for installing Confluent Cloud CLI and configuring it to your Confluent Cloud environment is available from within the Confluent Cloud Console: navigate to your Kafka cluster, click on `Dashboard` from the list on the left, then click the `Set up the CLI` link, and run through the steps in the `Set up CLI` page.
```

Comment:
1. I don't see what you see (Dashboard, Set up CLI, etc) ;) 
2. I think the CLI has been renamed from `Confluent Cloud CLI` to just `Confluent CLI` -- please confirm, but so, then the text should change here and throughout all KT headings (in the harness files)